Duplicati/Library/Utility/CallContextSettings.cs

```diff
@@ -236,7 +236,7 @@ public static class CallContextSettings<T>
         /// The context setting types that are used and need their private setting key
         /// to prevent overwriting each others settings.
         /// </summary>
-        private static string contextSettingsType = null;
+        private static string? contextSettingsType = null;

```

Comment:
This should not be nullable, because it can always be initialized in the constructor.

Reply:
Fixed this